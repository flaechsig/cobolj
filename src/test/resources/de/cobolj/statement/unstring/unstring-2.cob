IDENTIFICATION DIVISION. 
    PROGRAM-ID. HELLO-WORLD.
    
DATA DIVISION.
WORKING-STORAGE SECTION.
  01 WS-QUELLE PIC X(25) VALUE '00001#Max#Muster'.
  01 WS-SENKE.
     05 WS-NUMMER PIC A(5).
     05 WS-VORNAME PIC A(10).
     05 WS-NACHNAME PIC A(10).
  
PROCEDURE DIVISION.
  UNSTRING WS-QUELLE
    DELIMITED BY '#'
    INTO WS-NUMMER WS-VORNAME WS-NACHNAME.
  DISPLAY WS-NUMMER.
  DISPLAY WS-VORNAME.
  DISPLAY WS-NACHNAME.
  DISPLAY WS-QUELLE.
  STOP RUN.