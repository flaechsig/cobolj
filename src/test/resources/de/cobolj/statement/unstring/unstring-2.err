Einfache Unstring-Anweisung mit doppelten Delimiter, aber ohne ALL