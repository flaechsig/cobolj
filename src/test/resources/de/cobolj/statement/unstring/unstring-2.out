00001
Max       
Muster    
00001#Max#Muster         
